### PR TITLE
Trace assembly version

### LINKED ISSUE #42: Print version flag: `--version`
Would be nice to have the expecto version print into first line on startup

### PR DESCRIPTION
**Pull request by forki**
closes #42

### CODE REVIEW

File: Expecto/Expecto.fs [hunk 1]
```diff
@@ -819,23 +819,31 @@ module Tests =
...
+        | Version -> "Prints out version information."
 
   [<CompilationRepresentation (CompilationRepresentationFlags.ModuleSuffix)>]
   module ExpectoConfig =
+    open System.Diagnostics
+
+    let printExpectoVersion() =
+      let assembly = Assembly.GetExecutingAssembly()
+      let fileInfoVersion = FileVersionInfo.GetVersionInfo assembly.Location
+      logger.info (eventX "EXPECTO version {version}" >> setField "version" fileInfoVersion.ProductVersion)
```
Comment (haf) [inline Expecto/Expecto.fs]: 80 chars

**Comment by haf** [comment]
Thank you. I would merge this if it fit the DEVGUIDE.

File: Expecto/Expecto.fs (hunk 1 above)
Comment (forki) [inline Expecto/Expecto.fs]: you should really buy yourself a modern monitor ;-)

**Comment by forki** [comment]
done

**Pull request merged**